subprojects/docs/src/docs/release/notes.md

```diff
@@ -63,6 +63,26 @@ users of the plugin and so should be used carefully.
 
 For more details see (user manual link)
 
+
+## Improvements for "ComponentMetadataRule"s
+
+The API for component metadata rules has been extended to allow access to `packaging`, `classifier` and `type` information from POM metadata.
+The `packaging` information for a direct dependency [can be obtained](userguide/component_metadata_rules.html#filter_using_maven_metadata) from the `ComponentMetadataContext`.
+Additionally, `ComponentMetadataRule`s can react on the `classifier` and `type` information of the transitive dependencies:
+```
+class ClassifierExampleRule implements ComponentMetadataRule {
```

Comment:
It would be nice if we would show a real complete (but small) use case here. But I can't think of one myself right now because the feature itself is mostly applicable for more complex cases.

Lets keep it like this.